### PR TITLE
Add latencyMode to VideoEncoderConfig

### PR DESCRIPTION
**Pull request by chcunningham**
Per plan described in #269. Will add framerate in a follow up.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webcodecs/pull/292.html" title="Last updated on Jul 14, 2021, 5:25 AM UTC (c405b93)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webcodecs/292/09f2655...c405b93.html" title="Last updated on Jul 14, 2021, 5:25 AM UTC (c405b93)">Diff</a>

### CODE REVIEW

**Review by aboba — APPROVED** [review]

**Comment by chcunningham** [comment]
@padenot friendly ping

File: index.src.html [hunk 1]
```diff
@@ -1901,6 +1907,29 @@
...
+enum LatencyMode {
+  "quality",
+  "realtime"
+};
+</xmp>
+
+: <dfn enum-value for=LatencyMode>quality</dfn>
+:: Indicates that the user agent should optimize for encoding quality. In
+    this mode:
+    * User agents may increase encoding latency to improve quality.
```
Comment (padenot) [inline index.src.html:1922]: `s/User agents/User Agents/`

Comment (padenot) [inline index.src.html:1922]: We'll want to do one pass to capitalize the rfc2119 words I think at some stage. There are lots of uses in non-normative section also.

**Review by padenot — APPROVED** [review]

File: index.src.html (hunk 1 above)
Comment (chcunningham) [reply index.src.html:1922]: Sg, filed #301

Comment (chcunningham) [reply index.src.html]: Done.

Comment (chcunningham) [reply index.src.html]: Filed #301

**Review by chcunningham — COMMENTED** [review]
Thanks!

**Pull request merged**